src/actions.js

```diff
@@ -19,3 +19,13 @@ export const REMOVE_CART_ITEM = 'REMOVE_CART_ITEM';
 export const TOGGLE_CART_ITEM_AMOUNT = 'TOGGLE_CART_ITEM_AMOUNT';
 export const CLEAR_CART = 'CLEAR_CART';
 export const COUNT_CART_TOTALS = 'COUNT_CART_TOTALS';
+export const LOGIN_BEGIN = 'LOGIN_BEGIN';
+export const LOGIN_SUCCESS = 'LOGIN_SUCCESS';
+export const LOGIN_ERROR = 'LOGIN_ERROR';
+export const REGISTER_BEGIN = 'REGISTER_BEGIN';
+export const REGISTER_SUCCESS = 'REGISTER_SUCCESS';
+export const REGISTER_ERROR = 'REGISTER_ERROR';
+export const LOGOUT_BEGIN = 'LOGOUT_BEGIN';
+export const LOGOUT_SUCCESS = 'LOGOUT_SUCCESS';
+export const LOGOUT_ERROR = 'LOGOUT_ERROR';
+export const CURRENT_USER_FETCHED = 'CURRENT_USER_FETCHED';
```

Comment:
These strings are referred to as "actions" when talking about reducers (same terminology is used for Redux). They're basically just names of custom 'events' that we are defining and triggering in our code. 

It is common convention to write these as all-capitals, just to indicate that these are constants that we don't expect to change or mutate in our code.